modules/backendworker/backendworker_test.go

```diff
@@ -72,7 +72,7 @@ func setupDependencies(ctx context.Context, t *testing.T, limits overrides.Confi
 	workerConfig.BackendSchedulerAddr = "localhost:1234"
 	workerConfig.Ring.KVStore.Store = "inmemory"
 	workerConfig.Ring.KVStore.Mock = nil
-	workerConfig.Ring.InstanceInterfaceNames = []string{"eth0", "en0", "lo0", "enp102s0f4u1u3"}
+	workerConfig.Ring.InstanceInterfaceNames = []string{"eth0", "en0", "lo0", "enp102s0f4u1u3", "eno1"}
```

Comment:
i'm guessing an empty slice doesn't check all interfaces? or maybe we could write a func that just grabbed all interfaces and added it to the config for tests?

fine with this if it just unblocks.

also lol at `enp102s0f4u1u3`

Reply:
Also you will need to push the merge button, I cant.